luarules/gadgets/unit_zombies.lua

```diff
@@ -139,28 +159,28 @@ local function calculateHealthRatio(featureID)
 			damagedReductionRatio = health/maxHealth
 		end
 	end
-	local result = (partialReclaimRatio + damagedReductionRatio) * 0.5 --average the two ratios to skew the result towards maximum health
-	return result
+	local healthRatio = (partialReclaimRatio + damagedReductionRatio) * 0.5 --average the two ratios to skew the result towards maximum health
+	return healthRatio
 end
 
 local function GetUnitNearestAlly(unitID, unitDefID, range)
-	local best_ally
-	local best_dist
+	local bestAllySq
+	local bestDistanceSq
```

Comment:
```suggestion
	local bestAlly
	local bestDistanceSquared
```
Best ally is a unit ID, not a distance

Reply:
done